test/testtask_test.rb

```diff
@@ -25,4 +26,12 @@ def test_define_rake_tasks
   def test_composing_rake_testtask
     assert_equal @rake_task, @ttnt_task.rake_testtask
   end
+
+  def test_expanded_file_list
+    # It gathers tests from both `pattern` and `test_files` option for Rake::TestTask
+    assert_equal(
```

Comment:
Cool, thank you! Fixed: https://github.com/Genki-S/ttnt/commit/8a0f17e6d6dd9912bea2c19e06e213d881c1f266